index.rst

```diff
@@ -0,0 +1,62 @@
+.. populationgenomics documentation master file, created by
+   sphinx-quickstart on Wed Mar 13 16:30:03 2024.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to populationgenomics's documentation!
+==============================================
+
+Welcome to the Centre for Population Genomics! We're glad to have you here :)
+
+This repository contains documentation you might find useful as you onboard, but and as an ongoing reference. We sometimes let these docs fall out of date, so please have a chat on Slack if something seems unclear - or open a pull request!
+
+If you need help more urgently, don't hesitate to ask directly on Slack (e.g. in
+the `#team-data` or `#software-questions` channels).
+
+
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :caption: Setup:
+
+   getting_started.md
+   terminal.md
+   zscaler.md
+   git.md
+   python.md
+   code_editors.md
+   notebooks.md
+   hail.md
+
+.. toctree::
+   :maxdepth: 1
+   :caption: About the centre
+
+   reproducible_analyses.md
+   storage_policies/README.md
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Libraries
+
+   cpg-utils/index
+   production-pipelines/index
```

Comment:
@vivbak, see here (+ the cpg-utils/index.rst) for how to link your new docs. I recommmend creating an index page for metamist, and then creating pages off it.